### DISCUSSION TITLE
Wrong URL of keycloak 15.2 device authorization endpoint

### DISCUSSION THREAD

**Discussion by BirahimFall** [keycloak/keycloak#8580] [Q&A]
I have reported an issue related to keycloak 15.2 device authorization endpoint.
The endpoint URI  "protocol/openid-connect/auth/device" while I believe it should "protocol/openid-connect/device/auth" as specified here: https://github.com/keycloak/keycloak-community/blob/master/design/oauth2-device-authorization-grant.md

https://github.com/keycloak/keycloak-community/issues/323

**Comment by stianst** [comment]
Please open an issue in JIRA to report a bug, see https://www.keycloak.org/community for more details
